public_src/components/beginner/beginner.component.html

```diff
@@ -2,7 +2,7 @@
   <i class="fa fa-caret-left  icon-2x" aria-hidden="true"> Back </i>
 </button>
 <accordion>
-  <accordion-group #a1 [isOpen]="true" *ngIf="!(errorCorrectionMode$ | async)">
+  <accordion-group #a1 [isOpen]="true" *ngIf="(errorCorrectionMode$ | async) === 'find errors' || (errorCorrectionMode$ | async) === 'menu' || (errorCorrectionMode$ | async) === null">
```

Comment:
I would prefer to put these more complex conditions into the separate function in the .ts file